pkgs/tools/text/shfmt/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "shfmt";
-  version = "3.9.0";
+  version = "3.10.0";
 
   src = fetchFromGitHub {
     owner = "mvdan";
     repo = "sh";
     rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
While we're manually updating.

---

No nitpicks, thanks.

---

We should just do a treewide with this

---

Either a treewide or get nix-update, nixpkgs-update, etc to add it.